apps/web/pages/event-types/[type].tsx

```diff
@@ -461,9 +473,7 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
     team ? `team/${team.slug}` : eventType.users[0].username
   }/${eventType.slug}`;
 
-  const placeholderHashedLink = `${process.env.NEXT_PUBLIC_WEBSITE_URL}/d/${
-    eventType.hashedLink ? eventType.hashedLink.link : "xxxxxxxxxxxxxxxxx"
-  }/${eventType.slug}`;
+  const placeholderHashedLink = `${process.env.NEXT_PUBLIC_WEBSITE_URL}/d/${hashedUrl}/${eventType.slug}`;
```

Comment:
URL on preview apps is still not of the preview app but it is cal.dev. 
@zomars maybe we can modify WEBSITE_URL in constants to be equivalent to VERCEL_URL for preview apps and then use that variable here?

```
export const WEBSITE_URL = VERCEL_URL.endsWith(".vercel.app") ? VERCEL_URL : process.env.NEXT_PUBLIC_WEBSITE_URL || "https://cal.com";
```